packages/common/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [2.7.0] - 2023-08-25
```

Comment:
better use `chan` 

missing head comparison at the bottle of changelog

`
[2.7.0]: https://github.com/subquery/subql/compare/common/2.6.0...common/2.7.0`